PyDSS/value_storage.py

```diff
@@ -262,67 +282,85 @@ def __init__(self, name, prop, value, Nodes, is_complex, units):
                 if is_complex:
                     label += " " + units[0]
                     self._labels.append(label)
-                    self._data[label] = [complex(x[0], x[1])]
+                    self._value += [complex(x[0], x[1])]
```

Comment:
@AadilLatif Can you confirm that the setting of self._value is correct here and below?